make-config.sh

```diff
@@ -2,39 +2,49 @@
 echo %source_type application/pdf
 echo %target_type application/pdf
 echo %shell_command $(pwd)/conversion-scripts/soffice-wrapper.sh -i pdf -o pdf
+echo %id soffice-pdf-to-pdfa
```

Comment:
I was totally going to suggest that you update `make-config.sh` w/ the id-s and you totally beat me to it!

Reply:
Yeah, realized that this morning. And yeah, we might as well put the file extensions in the config file while we're at it. I can put that on the docket.